examples/build_overrides/tizen.gni

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2021 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare_args() {
+  # Root directory for tizen.
+  tizen_root = "//config/tizen/chip-gn/platform"
+  tizen_home = ""
```

Comment:
It seems rather confusing to have both tizen_root and tizen_home.

Reply:
I changed the name to tizen_sdk. but, I can't change the env TIZEN_HOME because already used in docker. I will change it together in later.